src/common/_modules/application_gateway/variables.tf

```diff
@@ -0,0 +1,120 @@
+variable "project" {
+  type        = string
+  description = "IO prefix, short environment and short location"
+}
+
+variable "prefix" {
+  type    = string
+  default = "io"
+  validation {
+    condition = (
+      length(var.prefix) < 6
+    )
+    error_message = "Max length is 6 chars."
+  }
+}
+
+variable "location" {
+  type        = string
+  description = "Azure region"
+}
+
+variable "location_short" {
+  type        = string
+  description = "Azure region short name"
+}
+
+variable "tags" {
+  type        = map(any)
+  description = "Resource tags"
+}
+
+variable "resource_groups" {
+  type        = map(string)
```

Comment:
I'm not sure about this variable structure. How can I know that a specific rg is there?

Reply:
Good point! The thinking behind the choice is based on the lack (yet) of the resource groups structure we want to adopt during the migration to italy north

I kept it extremely free for this reason, but we can surely find a structure that can accomodate both regions and structures. Maybe changes in the usage of resource groups may be inconvenient, but I think it deserves a discussion